src/components/icons/Icon.stories.ts

```diff
@@ -0,0 +1,38 @@
+import { type Meta, type StoryFn } from '@storybook/vue';
+import Icon from './Icon.vue';
+
+const render: StoryFn<typeof Icon> = (_, { argTypes }) => ({
+  components: { Icon },
+  props: Object.keys(argTypes),
+  template: '<Icon v-bind="$props" class="text-rui-primary" />',
+});
+
+const meta: Meta<typeof Icon> = {
+  title: 'Components/Icon',
+  component: Icon,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'All icons can be seen here: <a target="_blank" href="https://remixicon.com/">https://remixicon.com/</a>. Use it <b>without</b> prefix `ri-`',
+      },
+    },
+  },
+  tags: ['autodocs'],
+  render,
+  argTypes: {
+    name: { control: 'text' },
+    size: {
+      control: 'number',
+    },
+  },
+};
+
+export const Primary = {
+  args: {
+    name: 'arrow-down-circle-fill',
```

Comment:
Btw would it be possible for the storybook to have the icon name field provide suggestions?

![image](https://github.com/rotki/ui-library/assets/2269732/802dca6c-e6f5-456f-90f5-628c58abbff6)